booklets_text/2013.txt

```diff
@@ -2315,9 +2315,9 @@ Project Title: GetToKnowMe
 Name: Danil Rusakov
 Email: danil.rusakov2@mail.dcu.ie
 Name: Alexandra Moldovan
-Email: alexandramonica.
-moldovan2@mail.dcu.ie
+Email: alexandramonica.moldovan2@mail.dcu.ie
 Name: Radoslav Kissiov
+Email: 
```

Comment:
The Email was missing for a couple of students (probably because it couldn't be fit in because there's 3 students). Just add this empty email field to make the parsing successful.